rllib/agents/sac/sac_policy.py

```diff
@@ -89,8 +92,11 @@ def postprocess_trajectory(policy,
 
 
 def get_dist_class(config, action_space):
-    action_dist_class = SquashedGaussian if \
-        config["normalize_actions"] is True else DiagGaussian
+    if isinstance(action_space, Discrete):
+        action_dist_class = Categorical
+    else:
+        action_dist_class = SquashedGaussian if \
+            config["normalize_actions"] is True else DiagGaussian
```

Comment:
No need for `is True` I believe.

```suggestion
        action_dist_class = (
            SquashedGaussian if config["normalize_actions"]
            else DiagGaussian)
```